infra/macos/scripts/rebuild-crontask.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
```suggestion
# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
#!/bin/bash
```

Reply:
I think the shebang has to go on the first line.

In _build.sh_, we use this:

```suggestion
#!/usr/bin/env bash
# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
# SPDX-License-Identifier: Apache-2.0
```

(Probably meaningful that we use `/usr/bin/env bash`, too. Mine, at least, comes from `$HOME/.nix-profile/bin/bash`.)

Reply:
At least on my machine, having a comment before the shebang works. Not sure if this is a macOS thing.

Re: `env`, yes, this is usually the preferred way to invoke bash when writing portable scripts. In this case, though, this script is meant to run within a VM we completely control (created by the rest of this folder), so I think using `/bin/bash` is fine.

Reply:
Silly me. Not fully awake yet it seems. Corrected shebang lines to be first.